exercises/practice/strain/strain-test.rkt

```diff
@@ -0,0 +1,80 @@
+#lang racket/base
+
+(require "strain.rkt")
+
+(module+ test
+  (require rackunit rackunit/text-ui)
+
+  (define (always-true _) #t)
+
+  (define (always-false _) #f)
+
+  (define (contains-5? lst) (member 5 lst))
+
+  (define (starts-with-z? str) (equal? (string-ref str 0) #\z))
```

Comment:
Minor nitpick I missed earlier.`equal?` works well enough, but since we know we're comparing characters, `char=?` broadcasts intent and might be a better fit here. Passing in a non-character like `3` will result in a contract violation, giving the caller a bit more information than just returning the `#f` from `equal?`.

Reply:
Oh, I didn't know about `char=?`. Thanks! 😄

Reply:
Oh, I didn't know about `char=?`. Thanks! 😄